scripts/check-version.js

```diff
@@ -1,82 +1,84 @@
 #!/usr/bin/env node
 /* eslint-env node */
 
-const readline = require('readline')
-const path = require('path')
-const childProc = require('child_process')
-const fs = require('fs')
-const pkg = require('../package.json')
+const readline = require("readline");
```

Comment:
hmmm this file is automatically styled, but the style is different than the rest of the app... 🤔

Reply:
i just hit save on this file and the same thing happened, I don't think the auto formatting is correctly setup when we open the editor from the root.

Reply:
i think an easy way to fix this, is make a copy of  `./packages/pwa/.prettierrc.yaml` and move it to root

Reply:
you mean the one in pwa folder is different? We would need one in the retail react app projects though right?

Reply:
Yea, I believe the auto formatting was done by Prettier and we probably want to keep the entire repo following the same code style. There is a format configuration in the PWA folder, we probably should also have an exact same copy of `.prettierrc.yaml` in the root, so that the code in `/scripts` are formatted like the rest of the repository.